src/lib/sidenav/sidenav.md

```diff
@@ -18,6 +18,13 @@ A sidenav container may contain one or two `<md-sidenav>` elements. When there a
 `<md-sidenav>` elements, each must be placed on a different side of the container.
 See the section on positioning below.
 
+`<md-drawer>` is a component that is similar to `<md-sidenav>`. 
```

Comment:
On second thought I'll just add some documentation on `md-drawer` later, for now lets not mention it since its the exact same thing as `md-sidenav`